9e/TTSLUA/extractTerrain.ttslua

```diff
@@ -256,19 +256,20 @@ function createDebugMat()
 end
 
 function deleteTable()
-
     --flexTable=getObjectFromGUID(flexTableGuid)
     local guidList = {
         "afc863","c8edca","393bf7","12c65e","f938a2","9f95fd","35b95f",
         "5af8f2","4ee1f2","bd69bd", "28865a", "9f95fd"
     }
+    local obj = {}
```

Comment:
I am unsure if this is really buying you anything. I don't normally define stuff like this. Just declare it inside the for loop.